config/database.yml

```diff
@@ -1,25 +1,25 @@
-# SQLite version 3.x
-#   gem install sqlite3
+# MySQL version 2.x
+#   gem install mysql2
 #
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
+#   Ensure the MySQL 2 gem is defined in your Gemfile
+#   gem 'mysql2'
 #
+
 default: &default
-  adapter: sqlite3
+  adapter: mysql2
   pool: 5
-  timeout: 5000
+  username: <%= ENV['DATABASE_USER'] || "dev" %>
+  password: <%= ENV['DATABASE_PASS'] || "dev" %>
+  host: 127.0.0.1
 
-development:
+test:
   <<: *default
-  database: data/db/development.sqlite3
+  database: predeploychecker_test
 
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
-test:
+development:
   <<: *default
-  database: data/db/test.sqlite3
+  database: predeploychecker_dev
 
 production:
   <<: *default
```

Comment:
The `<<: *default` should be removed from here when using the `url` because you cannot define both individual values for `host`, `database`, etc. and `url`.